### DISCUSSION TITLE
Mountain Duck File Attribute Preservation

### DISCUSSION THREAD

**Discussion by hughes-james** [iterate-ch/cyberduck#15432] [Q&A] [answered]
I suspect that the answer is 'no', so I'm asking this more as a sanity check. I'm syncing a Wasabi bucket to my Mac using Mountain Duck and cannot see a way of preserving file attributes (creation & modified dates). I know that S3 is not a file system and that the modified date reported on a object is (generally) the upload date. Finder will subsequently report the modified date as the date is syncs the file down. If there any way to preserve these attributes for files that are synced up and then down (across several computers) or is this just how it is?

Very many thanks!

James

**Comment by dkocher** [comment, marked as answer]
You can make it work using the _S3 (Timestamps)_ connection profile available from _Preferences → Profiles_ which will store the modification date in the metadata of the S3 object. This should make it work, albeit with heavy performance costs for the additional HTTP requests required.

**Reply by hughes-james** [reply]
That's great, very many thanks!
